### ISSUE TITLE
Not getting a correct elevation value

### ISSUE THREAD

**Issue by slouque** [Esri/collector-tools#88]
Yesterday morning I took the receiver out to where one of our elevation monuments is located and collected a point. Came back into the office and ran the Project Z tool on the feature class containing the point I went collect. Plus the Project tool and Add XY Coordinates tool which the tutorial I followed said. It gave me a value very close to what the last calibrated reading for that monument was back in 2013. The reading on this monument from 2013 was 19.39 ft and the reading I got yesterday morning was 19.92 ft

Yesterday evening I went out again to the same monument to collect another point which I ran through the same process (I think) and I got a totally different elevation value than what I got in the morning. 25 ft

So this morning I went to another area to try again and am still getting a different reading than what I initially got.

My steps are as follows.

First I open up my web map in ArcGIS Pro and export the hosted feature class out to a file geodatabase feature class
Then I bring that feature class into the Project Z tool

![image](https://user-images.githubusercontent.com/62031277/82051687-f72a9f00-967f-11ea-85de-bbe28c331f5a.png)

Then once get that value I use the Project tool to project into a coordinate system we use here and then transform it into a NAVD88 (height)(USft)

![image](https://user-images.githubusercontent.com/62031277/82052103-b1baa180-9680-11ea-88d9-f8f19f09e4d8.png)

Once I have an elevation value, I then use the Add XY Coordinates tool to convert it to ft

![image](https://user-images.githubusercontent.com/62031277/82052171-d0b93380-9680-11ea-91e9-165ff528625b.png)

Am I missing a step?

I had it working what seemed perfect before

Thanks,

Scott Louque
St James Parish GIS Analyst

**Comment by slouque** [comment]
I think I solved my own problem! 

I was going over all my location settings in Collector and noticed the antenna height had defaulted back to 0 when it should be set at 2 m which makes sense cause all the values I was getting were 2 meters off from where they should be. So I set it back to 2 m and went collect another point and the value I got this time was more in line with what it should be for that location.

Thanks
